content/en/docs/concepts/services-networking/service.md

```diff
@@ -74,7 +74,7 @@ a new instance.
 The name of a Service object must be a valid
 [DNS label name](/docs/concepts/overview/working-with-objects/names#dns-label-names).
 
-For example, suppose you have a set of Pods that each listen on TCP port 9376
+For example, suppose you have a set of Pods that each listens on TCP port 9376
```

Comment:
Hi @hezhizhen .
What do you think about updating the sentence to:
For example, suppose you have a set of Pods where each Pod listens on TCP port 9376 and contains the
label `app=MyApp`:

Reply:
@kbhawkey What about now?